### PR TITLE
Add UserSid to report_event

### PR DESCRIPTION
**Pull request by api-walker**
I need to log the user to the MS Windows Event Log. I added the possibility to give a UserSid to the report_event function. It accepts a SID structure, which could be used from Win32::Security::SID.open('api-walker').sid for example.

### CODE REVIEW

File: lib/win32/eventlog.rb [hunk 1]
```diff
@@ -701,12 +703,18 @@ def report_event(args)
         num_strings = 0
       end
 
+      if hash['user_sid']
+        user_sid = FFI::MemoryPointer.from_string(hash['user_sid'])
```
Comment (api-walker) [inline lib/win32/eventlog.rb]: Great!

Comment (djberg96) [inline lib/win32/eventlog.rb]: @api-walker Ok, go ahead and push that change, then I'll merge.

**Comment by api-walker** [comment]
I pushed the changes.

**Comment by djberg96** [comment]
Released 0.6.7. :)

**Pull request merged**